src/project/projecttypes.h

```diff
@@ -88,6 +89,30 @@ struct CloudProjectInfo {
     }
 };
 
+struct ExportInfo {
+    ExportType type;
```

Comment:
Do we really need to store a copy of the whole ExportType? Or is storing the ID enough? (Note the existence of the method `ExportDialogModel::selectExportTypeById`)